rust-runtime/protocol-test-helpers/src/lib.rs

```diff
@@ -341,9 +341,12 @@ fn rewrite_url_encoded_map_keys(input: &str) -> (String, String) {
 fn rewrite_url_encoded_body(input: &str) -> String {
     let mut entries: Vec<(String, String)> = input
         .split("\n&")
+        .filter(|s| !s.is_empty())
```

Comment:
not sure of the exact context, but you might want `s.trim().is_empty()`

Reply:
Fixed